beancount_import/source/schwab_csv.py

```diff
@@ -885,6 +1053,10 @@ def _get_key_from_posting(
             return None
         if not posting.account in account_set:
             return None
+        if not POSTING_META_ACTION_KEY in posting.meta:
+            return None
+        if not POSTING_DATE_KEY in posting.meta:
+            return None
```

Comment:
If we do leave this in, I'd rather handle it the same as `source_desc` (get the value with an empty default and return `None` if it is empty, rather than checking if the key is there and then later getting the value for that key.)